package.json

```diff
@@ -163,9 +163,8 @@
     "@types/react-modal": "^3.10.1",
     "@types/react-select": "^3.0.8",
     "@types/styled-components": "4.1.8",
-    "@typescript-eslint/eslint-plugin": "^1.13.0",
-    "@typescript-eslint/parser": "^1.13.0",
-    "acorn": "^7.1.0",
```

Comment:
This was required at some point as peer dependency of a sub-sub-dependency of `@typescript-eslint` but it's not anymore with the upgrade.